src/app/frontend/replicasetdetail/replicasetdetail.html

```diff
@@ -14,27 +14,145 @@
 limitations under the License.
 -->
 
-<div layout="column" layout-margin layout-align="center center">
-  Displaying details for replica set {{ctrl.replicaSetDetail.name}} in namespace
-  {{ctrl.replicaSetDetail.namespace}}.
-  <div>
-    <table>
-      <thead>
-        <tr>
-          <th>Pod</th>
-          <th>Start time</th>
-          <th>IP</th>
-          <th>Node</th>
-        </tr>
-      </thead>
-      <tbody>
-        <tr ng-repeat="pod in ctrl.replicaSetDetail.pods">
-          <td>{{pod.name}}</td>
-          <td>{{pod.startTime}}</td>
-          <td>{{pod.podIP}}</td>
-          <td>{{pod.nodeName}}</td>
-        </tr>
-      </tbody>
-    </table>
+<md-content layout="row" layout-fill flex>
+  <div class="kd-replicasetdetail-sidebar" layout="column">
+    <div flex layout-align="start center" layout="row" class="kd-header-option-panel">
+      <a flex-offset="10" ui-sref="replicasets">
+        <i class="material-icons kd-align-mid kd-font-lg">keyboard_backspace</i>
+      </a>
+      <span flex-offset="5" flex class="kd-align-mid md-title">
+        {{ctrl.replicaSetDetail.name}}
+      </span>
+    </div>
+    <div flex-offset="5" flex layout-align="start start" layout-padding layout="row">
+      <div flex layout="column">
+        <h3 layout="row" class="kd-primary md-body-2">
+          <div flex="55">
+            <i class="kd-align-mid material-icons">refresh</i>
+            <span class="kd-align-mid">ROLLING UPDATE</span>
+          </div>
+          <div flex-offset="5" flex>
+            <i class="kd-align-mid material-icons">delete</i>
+            <span class="kd-align-mid">DELETE</span>
+          </div>
+        </h3>
+        <h1 class="md-body-2">
+            <span class="kd-align-mid">
+              {{ctrl.replicaSetDetail.pods.length}} pods
+            </span>
+          <i class="kd-editpen-md material-icons kd-primary kd-align-mid">
+            mode_edit
+          </i>
+        </h1>
+        <h1 class="md-caption kd-font-1">Label selector</h1>
+        <h1 class="md-caption kd-font-2">
+          <b>{{ctrl.formatLabelString(ctrl.replicaSetDetail.labelSelector)}}</b>
+        </h1>
+        <h1 class="md-body-2">Replica Sets</h1>
+        <h1 class="md-caption kd-font-1">Labels</h1>
```

Comment:
I see that you're using h1-s and apply class md-caption to every row. Instead, it is better to place your visual elements into semantic boxes. For example:
```html
<div class="my-detail">
   <span class="my-detail-title">Label selector</span>
  <span class="my-detail-line">{{ctrl.formatLabelString(ctrl.replicaSetDetail.labelSelector)}}</span>
</div>
```
... and repeat this for all parts. 

With this you dont tie yourself to implementation details ("kd-font-1"). You just say what things are in real life and then style them in css.